package.json

```diff
@@ -106,7 +106,7 @@
     "@openapitools/openapi-generator-cli": "1.0.13-4.3.1",
     "@types/file-saver": "2.0.1",
     "@types/jest": "26.0.20",
-    "@types/node": "13.13.4",
+    "@types/node": "14.14.20",
```

Comment:
Does this affect any already developed code?

Reply:
Nop